velox/exec/tests/OperatorUtilsTest.cpp

```diff
@@ -206,3 +207,56 @@ TEST_F(OperatorUtilsTest, gatherCopy) {
 TEST_F(OperatorUtilsTest, makeOperatorSpillPath) {
   EXPECT_EQ("spill/task_1_100", makeOperatorSpillPath("spill", "task", 1, 100));
 }
+
+TEST_F(OperatorUtilsTest, wrap) {
+  auto rowType = ROW({
+      {"bool_val", BOOLEAN()},
+      {"tiny_val", TINYINT()},
+      {"small_val", SMALLINT()},
+      {"int_val", INTEGER()},
+      {"long_val", BIGINT()},
+      {"ordinal", BIGINT()},
+      {"float_val", REAL()},
+      {"double_val", DOUBLE()},
+      {"string_val", VARCHAR()},
+      {"array_val", ARRAY(VARCHAR())},
+      {"struct_val", ROW({{"s_int", INTEGER()}, {"s_array", ARRAY(REAL())}})},
+      {"map_val",
+       MAP(VARCHAR(),
+           MAP(BIGINT(),
+               ROW({{"s2_int", INTEGER()}, {"s2_string", VARCHAR()}})))},
+  });
+
+  for (int32_t iter = 0; iter < 20; ++iter) {
```

Comment:
Why do we need this loop? We are not using a seed with the fuzzer, hence, it will generate same vector on each iteration, no?

```
  VectorFuzzer(
      VectorFuzzer::Options options,
      memory::MemoryPool* pool,
      size_t seed = 123456)
      : opts_(options), pool_(pool), rng_(seed) {}
```

Reply:
The loop is not randomize the fuzzer vector but randomize the indices.

Reply:
Got it. Thank you for explaining. The vector can be created outside of the loop then, no?

Reply:
Yeah :)